osf/management/commands/migrate_pagecounter_data.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from framework import sentry
+
+logger = logging.getLogger(__name__)
+
+
+LIMIT_CLAUSE = ' LIMIT %s);'
+NO_LIMIT_CLAUSE = ');'
```

Comment:
I don't love this, but I can't think of a different way to do it, to make the LIMIT dynamic, and the page_size dynamic.